docs/agents/browser.md

```diff
@@ -0,0 +1,123 @@
+title: Browser SDK
+description: Ship browser metrics to Sematext Experience.
+
+Sematext [Browser SDK](https://github.com/sematext/browser-sdk) is an open-source framework used to ship web application metrics related to user interaction. Using [Browser SDK](https://github.com/sematext/browser-sdk) allows gathering metrics related to:
+ 
+ - Hard page loads for traditional web applications as well as the single-page applications
+ - Soft page loads generated via the `routeChange` command for the single-page applications
+ - HTTP requests
+ - [Web application resources, such as images, CSS files, etc.](/experience/resources/)
+ - [Web vitals](/experience/webvitals/)
+ - [Element timing](/experience/element-timing/)
+ - [Long tasks](/experience/longtasks/)
+ - [User sessions](/experience/user-identification/)
+ - [On-page transactions](/experience/on-page-transaction/)
+
+![Sematext Experience Overview](../images/experience/overview.png)
+
+## Development
+
+The RUM script is developed using [ECMAScript 2015](https://en.wikipedia.org/wiki/ECMAScript) and uses a few tools to help with the build and working with the code. Those tools include:
+ 
+ - `npm` - [Node package manager](https://npm.org)
+ - `yarn` - [Package dependency manager](https://yarnpkg.com)
+ - `flow` - [Static type checker](https://flow.org)
+ - `Cypress.io` - [Integration tests framework](https://www.cypress.io)
+
+### The Architecture of the RUM Script
+
+There are two parts of the RUM script: 
+
+ - The loader script,
+ - The metrics gathering script itself.
+
+The loader script is responsible for loading the main, metrics gathering script itself and adds the actual `<script>` tags with the content on the main metric script. 
+
+The loader script will keep track and buffer any of the commands that need to be sent after the main metrics gathering script will be loaded and configured. When the main RUM script finally loads it will read any pending
```

Comment:
> The loader script will keep track and buffer any of the commands

The loader script keeps track and buffers commands 

> the main metrics gathering script will be loaded and configured

the metrics gathering script loads and is configured

> When the main RUM script finally loads it will read any pending

What's "main RUM script"?  Is it the metrics gathering script?  If so, let's refer to it with the same name consistently.

And let's not go to future tense and stick to present simple tense :)

Reply:
Done.